nodejs/awsx/ecs/ec2Service.ts

```diff
@@ -33,7 +33,6 @@ export class EC2TaskDefinition extends ecs.TaskDefinition {
             ...args,
             containers,
             requiresCompatibilities: ["EC2"],
-            networkMode: utils.ifUndefined(args.networkMode, "awsvpc"),
```

Comment:
this will certainly break things for people.  i think the better way would be to simply doc that this is the default behavior and you should pass `"networkMode: null` if you don't want a network mode provided on your behalf.